go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -110,7 +110,11 @@ func main() {
 	ctx := context.Background()
 	ctx, cancel := context.WithCancel(ctx)
 	c.Action = func(ctx *cli.Context) error {
```

Comment:
my understanding is `c.Action` get's triggered via 
```
	if err := c.RunContext(ctx, os.Args); err != nil {
		klog.Exit(err)
	}
```
where we print the error and exit.
@kyrtapz will look into why we don't print the error there and the weirdness its causing.
but this fix is not necessarily wrong - just want to understand why the existing klog.Exit doesn't work.